### DISCUSSION TITLE
Unblock chrome://extensions

### DISCUSSION THREAD

**Discussion by schoolexploitkid** [3kh0/ext-remover#1389] [Exploits JOIN TITANIUM NETWORK TO SHARE PLEASE]
Don't actually do this!!! I'm not responsible for stuff that happens to ur cb.

1. Start violently banging ur chromebook on the table for 30 seconds
2. Open it up and pray that it works
3. If it broke, womp womp to u ur screwed

this is from random person in TN discord

**Comment by schoolexploitkid** [comment]
@S-PScripts tangible 
also pls upvote
(watch me get ban)

**Comment by akabutnicer** [comment]
this isn't 2022-2023 anymore
